gluster-block-basic.yml

```diff
@@ -0,0 +1,56 @@
+- job:
+    name: gluster-block-basic
```

Comment:
I think the name is not apt. We should name it as gluster-block and rename the existing gluster-block.yml file to same as the job name i.e gluster_block_glusto. @nigelbabu What would you suggest?

Reply:
The beginning should always be `gluster_` for the name. I'm okay with `gluster_block_basic` or `gluster_block-basic`

Reply:
I personally think this test should be 'gluster_block' and the other one should be moved as gluster_block-glusto.. but for now it is OK.

Will do the changes!

Reply:
Done